### ISSUE TITLE
send property problem

### ISSUE THREAD

**Issue by fzn0x** [2pai/instastory-monitor-telegram#17]
Looks like send property has global properties for all dispatchers.

when it dones in one dispatcher, others dispatchers cannot send the media.
```js
if (story.send) return
```

How about creating a dispatcher send property for itself like :
```js
if (story.send_discord) return
```
and
```js
if (story.send_telegram) return
```

maybe the send property may be separated for several social media.

**Comment by 2pai** [comment]
Thanks for noticing, sure. 
Feel free to open a PR.

**Comment by fzn0x** [comment]
> Thanks for noticing, sure.
> Feel free to open a PR.

I'll do :D

**Comment by 2pai** [comment]
closed #18
